### DISCUSSION TITLE
Getting a 403 error

### DISCUSSION THREAD

**Discussion by D-Lite** [wavezync/nuxt-vue3-google-signin#2] [Q&A]
Following the setup for the Nuxt3, I consistently ran into this error
`[GSI_LOGGER]: The given client ID is not found.`

A deep dive into the network tab showed the client_id is returning undefined as part of the headers sent.

My assumption is that, the blocker is coming from this line in the nuxt.config.ts
` googleSignIn: {
    clientId: 'CLIENT ID OBTAINED FROM GOOGLE API CONSOLE',
  }`
  
  @kasvith

**Comment by kasvith** [comment]
Hi @D-Lite 

can you show me the nuxt.config

**Comment by D-Lite** [comment]
![nuxtconfigerr](https://user-images.githubusercontent.com/44473671/191697342-2df60336-3a3b-4059-8b19-bdb2c1592d74.png)

@kasvith

**Reply by kasvith** [reply]
thanks, i will create a fix

**Comment by D-Lite** [comment]
Great, 
Thanks (battled with this since yesterday before thinking of discussing here)

**Reply by kasvith** [reply]
I made a new release, please try it

**Comment by D-Lite** [comment]
Ok, will do so

**Reply by kasvith** [reply]
Can you confirm about the fix :)

**Comment by D-Lite** [comment]
![err](https://user-images.githubusercontent.com/44473671/191917082-4c691152-1b5c-4d45-a380-0d07f25a686c.png)

Same error

**Reply by kasvith** [reply]
can you give me a reproducible example? maybe in stackbliz or codesandbox?

**Reply by kasvith** [reply]
https://github.com/kasvith/nuxt-vue3-google-signin-example I ve created an example. this seems working fine.

Are you sure that client ID is working fine?

**Reply by kasvith** [reply]
You should use env variable like this, with `NUXT_PUBLIC` prefix i guess

 `process.env.NUXT_PUBLIC_GOOGLE_CLIENT_ID`

**Comment by D-Lite** [comment]
Great, working well now!

Thank you for the quick fix

**Comment by D-Lite** [comment]
Incase you're open to contributors, will love to contribute for future issues 

Cheers

**Reply by kasvith** [reply]
We would love to have contributors

**Reply by D-Lite** [reply]
Great stuff, will be open to meeting and having an overview
